src/helpers/helpers.time.js

```diff
@@ -55,8 +55,20 @@ module.exports = function(Chart) {
 		var ticks = [];
 		if (options.maxTicks) {
 			var stepSize = options.stepSize;
-			ticks.push(options.min !== undefined ? options.min : niceRange.min);
-			var cur = moment(niceRange.min);
+			var startTick = options.min !== undefined ? options.min : niceRange.min;
+			var majorUnitStart = startTick;
+			if (options.majorUnit) {
```

Comment:
`options.majorUnit` is used quite often in this method, I would cache it (`var majorUnit = options.majorUnit`) to increase minification.